src/odemis/gui/win/acquisition.py

```diff
@@ -783,6 +784,10 @@ def __init__(self, parent, orig_tab_data):
         self.btn_secom_acquire.Bind(wx.EVT_BUTTON, self.on_acquire)
         self.Bind(wx.EVT_CLOSE, self.on_close)
 
+        # range for the autofocus
+        self.autofocus_dist.SetValueRange(50e-6, 1000e-6)       # in m
```

Comment:
Maybe good to make sure that the distance does not exceed the selected area for the overview acquisition

Reply:
good suggetion, done

Reply:
can you also extend the comment to indicate what the values 50e-6 and 1000e-6 are based on?